guides/source/active_record_validations.md

```diff
@@ -810,7 +810,7 @@ class Person < ApplicationRecord
       # object = person object being validated
       # data = { model: "Person", attribute: "Username", value: <username> }
       message: ->(object, data) do
-        "Hey #{object.name}!, #{data[:value]} is taken already! Try again #{Time.zone.tomorrow}"
+        "Hey #{object.name}!, #{data[:value]} is taken already! Try again on #{Time.zone.tomorrow}"
```

Comment:
This makes the message grammatically correct, but the example is confusing. Why are we mentioning the date if it's not part of the validation? I'd remove the second sentence entirely.

Reply:
I agree, let me remove it.